content/en/cosign/signing_with_containers.md

```diff
@@ -1,17 +1,26 @@
 ---
-title: "Signing"
+title: "Signing Containers"
 category: "Cosign"
 position: 105
 ---
 
+You can use Cosign to sign containers with ephemeral keys by authenticating with an OIDC (OpenID Connect) protocol supported by Sigstore. Currently, you can authenticate with Google, GitHub, or Microsoft. For more information, read [Keyless Signatures](/cosign/keyless/). 
```

Comment:
This says "ephemeral keys" but earlier it said "keyless mode." Are these different or being used to refer to the same thing?